docs/application/native/guides/complications/watchface-complication.md

```diff
@@ -10,12 +10,12 @@ There are two ways to update the complication data.
 1. Watch Face can request to update complication data to complication provider application. The complication provider application receives the request and update the complication data. Then, the complication update callback is called in watch face.

```

Comment:
Suggested edit:

Watch Face can request complication provider application to update complication data. The complication provider application receives the request and updates the complication data. Then, the complication update callback is called in watch face.


Kindly check for technical accuracy before incorporating the comment.